sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -279,6 +279,7 @@ export class ReceiverImpl<ReceivedMessageT extends ReceivedMessage | ReceivedMes
    * Returns a promise that resolves to an array of messages based on given count and timeout over
    * an AMQP receiver link from a Queue/Subscription.
    *
+   * `timeout` provided in the `retryOptions` is not applicable while receiving the messages with receiveBatch.
```

Comment:
```suggestion
   * The `maxWaitTimeSeconds` provided via the options overrides the `timeoutInMs` provided in the `retryOptions`.
```

Reply:
Also, the `maxWaitTimeSeconds` need to be updated to be in terms of milliseconds as per Track 2 guidelines

Reply:
Same in other places